### ISSUE TITLE
Alpha hexadecimals and other directives.

### ISSUE THREAD

**Issue by LiquidAppleJuice** [wizzomafizzo/starcheat#261]
Starcheat doesn't seem to account for the alpha values at the end of directives. (?replace=ffffff00, for example would be a white with 0 opacity.)
Along with this, it doesn't seem to detect masks on headware or any other image directives aside from replace.

Link to other directives: http://pastebin.com/5pvg3Gth

**Comment by wizzomafizzo** [comment]
thanks for reporting this. is there anything in there that actually breaks the player or is it just cosmetic stuff in starcheat? this is a fairly large job, probably need to prioritise anything that breaks stuff

**Comment by Xirukai** [comment]
It does detect other directives.
The way to implement them has been altered. As for masks which govern what will be visible of your character. It does what it says on the tin, simply putting " " or [ ] cant remember which and you're good to go.

All depends on whether Starbound will reset values given or not. Since hardcoded items tend to be a pain, tried putting mframes of something onto an originally femaleFrames part of an item. Didnt work, altered to default.

Here's my code for making everything on the steamspring hat invisible. Except for the glasses part. It is working for me.

http://pastebin.com/VsWY4Fry

**Comment by LiquidAppleJuice** [comment]
Yeah, it's mainly cosmetic. It's not too big of an issue/bug, more just something that'd be helpful for gauging clothing edits.
(@Xirukai, it's the starcheat preview, not ingame itself. [This ](https://i.imgur.com/HM52dfr.png)is what happens if you try to view your changes in starcheat)

**Comment by Xirukai** [comment]
Ah yes. Okay.
